acme/api/handler.go

```diff
@@ -89,6 +91,11 @@ func (h *Handler) GetDirectory(w http.ResponseWriter, r *http.Request) {
 	api.JSON(w, dir)
 }
 
+// NotImplemented returns a 501. This is a place holder for future functionality.
```

Comment:
more precisely: _Semantically, this communicates a placeholder for future functionality._